### PR TITLE
dcp-911 project grouping filtering by biological network

### PR DESCRIPTION
**Pull request by Mil-m**
dcp-911
ebi-ait/dcp-ingest-central#911

### CODE REVIEW

**Comment by amnonkhen** [comment]
Please pay attention to PR and branch naming conventions.
They need to start with `dcp-`, for example dcp-911. This is how the automatic linking from branches, commits & PRs to issues and tickets works.

File: src/integration/java/org/humancellatlas/ingest/project/ProjectFilterTest.java [hunk 1]
```diff
@@ -450,6 +471,7 @@ void all_args_constructor() {
                 1,
                 DataAccessTypes.MANAGED,
                 "a label",
+                "",
```
Comment (amnonkhen) [inline src/integration/java/org/humancellatlas/ingest/project/ProjectFilterTest.java:474]: Don't use empty strings as parameters. Use some generic network name instead.

**Review by amnonkhen — CHANGES_REQUESTED** [review]

File: src/integration/java/org/humancellatlas/ingest/project/ProjectFilterTest.java (hunk 1 above)
Comment (Mil-m) [reply src/integration/java/org/humancellatlas/ingest/project/ProjectFilterTest.java:474]: like - 'a network'?

**Comment by Mil-m** [comment]
> Please pay attention to PR and branch naming conventions. They need to start with `dcp-`, for example [dcp-911](https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/911). This is how the automatic linking from branches, commits & PRs to issues and tickets works.

as I see we have 'dcp-911' both in the name of this branch 911-project-grouping-filtering-by-biological-network and in the commit names also

**Comment by amnonkhen** [comment]
@Mil-m have you noticed the tests are failing on gitlab?
image
Have you ran the tests locally?

**Review by amnonkhen — APPROVED** [review]

**Pull request merged**